int32 fan1Speed
int32 fan1Temp
int32 fan2Speed
int32 fan2Temp